student.rb

```diff
@@ -1,9 +1,12 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(id, classroom, age: nil, parent_permission: true, name: 'Unknown')
     super(id, age: age, parent_permission: parent_permission, name: name)
     @classroom = classroom
+    classroom&.add_student(self)

```

Comment:
this method is giving errors from the Linters it prefers the &.
 Style/SafeNavigation: Use safe navigation (&.) instead of checking if an object exists before calling the method.
    classroom.add_student(self) unless classroom.nil?